catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/view/Association.view.js

```diff
@@ -38,8 +38,21 @@ define([
             events: {
                 "click .remove-association": 'removeAssociation'
             },
+            initialize: function(options) {
+                this.readOnly = options.readOnly;
```

Comment:
You're right. For consistency, the same edit modal is used to display the remote metacard information as the editable local information. The read only stuff is used to prevent the user from making any changes in the modal.